src/main/resources/db/migration/changes/008-Instructors-create.json

```
@@ -0,0 +1,38 @@
+{
+  "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "Instructor-1",
+        "author": "wsong",
```

Please change these as follows (you can just click "commit suggestion" if you approve:

```suggestion
        "id": "008-Instructors-create",
        "author": "Wendy192837",
```